bokehjs/src/lib/models/axes/axis.ts

```diff
@@ -139,7 +140,7 @@ export class AxisView extends GuideRendererView {
 
   protected _axis_label_extent(): number {
     const text = this.model.axis_label
-    if (!text)
+    if (!text || text instanceof MathText)
```

Comment:
This is wrong. This method computes the extent (width or height) of a label, regardless what kind of label an axis is working with. Otherwise you're breaking basic design assumptions about this component, i.e. if an axis has a non-empty tex math label, this method must return a non-zero value.